<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Norbert Szabo</author>
 <class>NCRDPreferencesDialogUI</class>
 <widget class="QDialog" name="NCRDPreferencesDialogUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>444</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Preferences</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Report editor</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="spacing">
        <number>3</number>
       </property>
       <property name="leftMargin">
        <number>6</number>
       </property>
       <property name="topMargin">
        <number>6</number>
       </property>
       <property name="rightMargin">
        <number>6</number>
       </property>
       <property name="bottomMargin">
        <number>6</number>
       </property>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Default font</string>
         </property>
         <layout class="QGridLayout" name="g339">
          <item row="0" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Family</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QFontComboBox" name="comboFontFamily"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Point size</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QComboBox" name="comboFontSize">
            <property name="editable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="fontTest">
            <property name="text">
             <string>AaBbZz</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Font sample</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Default grid</string>
         </property>
         <layout class="QGridLayout" name="g554">
          <item row="0" column="0">
           <widget class="QCheckBox" name="cbGridVisible">
            <property name="text">
             <string>Visible</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Grid X</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QSpinBox" name="spGridX"/>
          </item>
          <item row="1" column="0">
           <widget class="QCheckBox" name="cbGridSnap">
            <property name="text">
             <string>Snap to grid</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Grid Y</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QSpinBox" name="spGridY"/>
          </item>
          <item row="1" column="1">
           <spacer name="sp001">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>10</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="5">
           <widget class="QPushButton" name="btnGridReset">
            <property name="text">
             <string>Reset grid</string>
            </property>
           </widget>
          </item>
          <item row="1" column="4">
           <spacer name="sp000">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_8">
         <property name="title">
          <string>Default item sizes</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="1">
           <widget class="QSpinBox" name="spFieldWidth">
            <property name="specialValueText">
             <string>-</string>
            </property>
            <property name="suffix">
             <string> px</string>
            </property>
            <property name="maximum">
             <number>500</number>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_14">
            <property name="text">
             <string>Field width</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_15">
            <property name="text">
             <string>Field height</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="label_16">
            <property name="text">
             <string>Field text</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="spFieldHeight">
            <property name="specialValueText">
             <string>-</string>
            </property>
            <property name="suffix">
             <string> px</string>
            </property>
            <property name="maximum">
             <number>500</number>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QLineEdit" name="fieldTxt"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>61</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Files and paths</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>3</number>
       </property>
       <property name="leftMargin">
        <number>6</number>
       </property>
       <property name="topMargin">
        <number>6</number>
       </property>
       <property name="rightMargin">
        <number>6</number>
       </property>
       <property name="bottomMargin">
        <number>6</number>
       </property>
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Default directories</string>
         </property>
         <layout class="QGridLayout" name="g549">
          <item row="0" column="2">
           <widget class="QToolButton" name="btnChooseDir">
            <property name="focusPolicy">
             <enum>Qt::NoFocus</enum>
            </property>
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Report directory:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="pathReport"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Output directory:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="pathOutput"/>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="pathTemplate"/>
          </item>
          <item row="1" column="2">
           <widget class="QToolButton" name="btnChooseOutputDir">
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Template directory:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QToolButton" name="btnChooseTemplateDir">
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_17">
            <property name="text">
             <string>Zint directory:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="pathZint"/>
          </item>
          <item row="3" column="2">
           <widget class="QToolButton" name="btnChooseZintDir">
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_5">
         <property name="title">
          <string>Counters</string>
         </property>
         <layout class="QGridLayout" name="g021">
          <property name="topMargin">
           <number>6</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Output file counter:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="spCounter">
            <property name="maximum">
             <number>999999</number>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_7">
         <property name="title">
          <string>HTML Output</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0" colspan="2">
           <widget class="QRadioButton" name="rbOneTableMode">
            <property name="text">
             <string>Sections build from one table</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="QRadioButton" name="rbMultiTableMode">
            <property name="text">
             <string>Sections build from multiple tables</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_13">
            <property name="text">
             <string>CSS file:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="cssFile"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Misc</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <property name="spacing">
        <number>3</number>
       </property>
       <property name="leftMargin">
        <number>6</number>
       </property>
       <property name="topMargin">
        <number>6</number>
       </property>
       <property name="rightMargin">
        <number>6</number>
       </property>
       <property name="bottomMargin">
        <number>6</number>
       </property>
       <item>
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string>Miscellaneous</string>
         </property>
         <layout class="QVBoxLayout" name="lay79811">
          <item>
           <widget class="QCheckBox" name="cbShowMaximized">
            <property name="text">
             <string>Open report windows maximized</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="cbEnc">
            <property name="text">
             <string>Encode stored database connection passwords</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_6">
         <property name="title">
          <string>Printer</string>
         </property>
         <layout class="QGridLayout" name="g408">
          <item row="0" column="0">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Printer resolution</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QComboBox" name="comboRes">
            <item>
             <property name="text">
              <string>High resolution</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Screen resolution</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>Default printer name</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0" colspan="3">
           <widget class="QCheckBox" name="cbNoDialog">
            <property name="text">
             <string>Print without printer dialog</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QLineEdit" name="lePrinterName"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>Printer DPI</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QSpinBox" name="spDPI">
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="specialValueText">
             <string>Auto</string>
            </property>
            <property name="minimum">
             <number>0</number>
            </property>
            <property name="maximum">
             <number>1200</number>
            </property>
            <property name="value">
             <number>0</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>210</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttons">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="3" margin="6"/>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>comboFontFamily</tabstop>
  <tabstop>comboFontSize</tabstop>
  <tabstop>fontTest</tabstop>
  <tabstop>cbGridVisible</tabstop>
  <tabstop>cbGridSnap</tabstop>
  <tabstop>spGridX</tabstop>
  <tabstop>spGridY</tabstop>
  <tabstop>btnGridReset</tabstop>
  <tabstop>spFieldWidth</tabstop>
  <tabstop>spFieldHeight</tabstop>
  <tabstop>fieldTxt</tabstop>
  <tabstop>cbShowMaximized</tabstop>
  <tabstop>comboRes</tabstop>
  <tabstop>spDPI</tabstop>
  <tabstop>lePrinterName</tabstop>
  <tabstop>cbNoDialog</tabstop>
  <tabstop>buttons</tabstop>
  <tabstop>btnChooseOutputDir</tabstop>
  <tabstop>pathReport</tabstop>
  <tabstop>btnChooseTemplateDir</tabstop>
  <tabstop>cbEnc</tabstop>
  <tabstop>pathOutput</tabstop>
  <tabstop>pathTemplate</tabstop>
  <tabstop>spCounter</tabstop>
  <tabstop>rbMultiTableMode</tabstop>
  <tabstop>cssFile</tabstop>
  <tabstop>rbOneTableMode</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttons</sender>
   <signal>accepted()</signal>
   <receiver>NCRDPreferencesDialogUI</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>507</x>
     <y>263</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttons</sender>
   <signal>rejected()</signal>
   <receiver>NCRDPreferencesDialogUI</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>507</x>
     <y>269</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
